pallets/pallet-bank/src/lib.rs

```diff
@@ -167,6 +232,9 @@ pub mod module {
 
         fn on_finalize(_block_number: BlockNumberFor<T>) {
```

Comment:
_name is used for when a variable isn't used.
If it's used, the _ should be removed.